build/locales/theater.de.js

```diff
@@ -0,0 +1 @@
+(function (w) { w.TheaterJS.prototype.keyboards.de = ["qwertzuiopü", "asdfghjklöä", "yxcvbnm"]; })(window);
```

Comment:
This file is supposed to be in the `src/locales/` folder. Gulp's job is to build the appropriate files automatically. If you put this file in `src/locales/` and run `gulp`, it'll create both `theater.de.js` and `theater.de.min.js`.